### ISSUE TITLE
Clean solution throws error when docker daemon is not running

### ISSUE THREAD

**Issue by kobellem** [microsoft/DockerTools#240]
We're working on a project with multiple developpers and recently added docker-compose integration. However we noticed when the docker daemon is not running, clean-solution throws the following error:

```
Error		Visual Studio Container Tools requires Docker to be running before building, debugging or running a containerized project.

For more info, please see: https://aka.ms/DockerToolsTroubleshooting	docker-compose	C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Sdks\Microsoft.Docker.Sdk\build\Microsoft.VisualStudio.Docker.Compose.targets	452	
```

This in contrast to the build command, which skips building the docker-compose project when the daemon is not running.

In our case the docker integration should be optional, a developer should be able to choose wether he wants to debug natively or in a docker container. Hence a request to align the clean command with the build command and not throw an error. Maybe it can be be brought down to Warning level.

Kind regards,
kobellem

**Comment by pratiksanglikar** [comment]
A fix for this issue is implemented internally, we will notify you when it becomes available for download in the latest release of Visual Studio.

**Comment by BinWuMSFT** [comment]
@kobellem This issue was fixed and released in Visual Studio 16.7. Could you please install and verify the fix?
